pkg/jtracer/jtracer.go

```diff
@@ -14,16 +14,57 @@
 
 package jtracer
 
-import "github.com/opentracing/opentracing-go"
+import (
+	"context"
+	"fmt"
+
+	"github.com/opentracing/opentracing-go"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
+	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
+	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
 
 type JTracer struct {
-	OT opentracing.Tracer
+	OT   opentracing.Tracer
+	OTEL trace.TracerProvider
 }
 
 func OT(t opentracing.Tracer) JTracer {
 	return JTracer{OT: t}
 }
 
+func OTEL(tp trace.TracerProvider) JTracer {
+	return JTracer{OTEL: tp}
+}
+
 func NoOp() JTracer {
 	return JTracer{OT: opentracing.NoopTracer{}}
 }
+
+func NoOpOtel() JTracer {
```

Comment:
So we certainly want to follow the principle of doing small PRs that perform one focused change. But this PR introduces changes like this function that is not actually used. Why do we need it? How do we know having this function is a good thing if we're not using it?

Reply:
I see. I made it in case we need it for future. For now, I don't see its use. I am going to revert it.